whitepaper.md

```diff
@@ -259,6 +260,17 @@ Profiling data produced by runtimes typically includes statistics down to the li
 
 > Image 2 shows an example icicle graph for a CPU profile of an application written in Go. It includes a kernel operation done by the Syscall call. This profile highlights that 35% of CPU time was used to compute hash, indicating potential optimization opportunities. [Source](https://pprof.me/9ce2c2d/).
 
+### Baggage
```

Comment:
I wonder, how is that a new signal vs trace detail? 🤔

Reply:
@bwplotka can you check https://opentelemetry.io/docs/concepts/signals/baggage/ ? From the documentation there, the `baggage` is a new signal at same level as  trace, metrics and logs.

Reply:
I agree with @bwplotka

Though Baggage is at the same level as traces, metrics, and logs on the site, one of the first paragraphs mentions that Baggage is transferred alongside Context which can be attached to traces, metrics, or logs. Given it attaches to those three signal types, Baggage isn't separate from them.